docs/table/modify_table.rst

```diff
@@ -164,14 +173,18 @@ the table and replaces existing ones::
    foo     1.0    10.0
    bar     2.0    10.0
 
-:meth:`~astropy.table.Table.update` also takes care of silently :ref:`merging_metadata`::
+``|`` and ``|=`` also takes care of silently :ref:`merging_metadata`::
 
   >>> t1.meta
   {'n': 2, 'id': 0}
 
-The input of :meth:`~astropy.table.Table.update` does not have to be a |Table|,
-it can be anything that can be used for :ref:`construct_table` with a
-compatible number of rows.
+The ``|`` and  ``|=`` operators are built on the
+:meth:`~astropy.table.Table.update` method, but the
```

Comment:
This is an implementation detail that should not be mentioned in the user documentation.